jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -35,6 +37,30 @@ def __init__(self, value, unit=None):
 @tray_registry('g-model-fitting', label="Model Fitting")
 class ModelFitting(PluginTemplateMixin, DatasetSelectMixin,
                    SpectralSubsetSelectMixin, AddResultsMixin):
+    """
+    See the :ref:`Model Fitting Plugin Documentation <specviz-model-fitting>` for more details.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``dataset`` (:class:`~jdaviz.core.template_mixin.DatasetSelect`):
+      Dataset to fit the model.
+    * ``spatial_subset`` (:class:`~jdaviz.core.template_mixin.SubsetSelect`):
+      Only exposed for Cubeviz.  Spatially collapsed spectrum to use to fit the model.
+    * ``spectral_subset`` (:class:`~jdaviz.core.template_mixin.SubsetSelect`)
+    * ``model_component`` (:class:`~jdaviz.core.template_mixin.SelectPluginComponent`)
+    * ``poly_order``
+    * ``model_component_label`` (:class:`~jdaviz.core.template_mixin.AutoLabel`)
+    * :meth:`create_model_component`
+    * ``equation`` (:class:`~jdaviz.core.template_mixin.AutoLabel`)
```

Comment:
this is a little ugly (that the class is called `AutoLabel` but being used for an equation, now that it is being exposed to the user).  I'm open to renaming the class, maybe something like `AutoTextField`?

Reply:
I don't think it's a big deal, but `AutoTextField` is a bit more general.

Reply:
ok, I think now is the time to do it since we haven't made it public yet.  I just pushed that change, but if anyone can think of a better name before merge, I'm happy to swap it out again (it's just a find and replace)!